schemas/app-new-form.json

```diff
@@ -25,6 +25,18 @@
           "maxLength": 0
         }
       ]
+    },
+    "clientId": {
+      "type": "string"
+    },
+    "allowGrantTypes": {
+      "type": "string"
+    },
+    "redirectUris": {
+      "type": "string"
+    },
+    "requirePkce": {
+      "type": "string"
```

Comment:
Can this be a `boolean`? There are a handful of places in this PR that it's typed as a `string`, but can it be coerced to a boolean at some point so the typing is more accurate?